### PR TITLE
UBERF-8895: Workspace UUIDs in PG/CR data tables

### PR DESCRIPTION
**Pull request by lexiv0re**
Adds new commands:

* generate-uuid-workspaces
* update-data-wsid-to-uuid

Note: both new commands expect only Mongo accounts

Related to: https://front.hc.engineering/workbench/platform/tracker/UBERF-8895

### CODE REVIEW

File: server/account/src/collections/mongo.ts [hunk 1]
```diff
@@ -178,6 +178,14 @@ export class WorkspaceMongoDbCollection extends MongoDbCollection<Workspace> imp
     super('workspace', db)
   }
 
+  async insertOne<K extends keyof Workspace>(data: Partial<Workspace>, idKey?: K): Promise<any> {
+    if (data.uuid === undefined) {
+      data.uuid = new UUID().toJSON()
```
Comment (haiodo) [inline server/account/src/collections/mongo.ts:183]: Why toJSON?

Comment (haiodo) [reply server/account/src/collections/mongo.ts:183]: Ok, stupid mongo.

**Review by haiodo — DISMISSED** [review]

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
@lexiv0re Lets @BykhovDenis will take a look as well :-)

File: dev/tool/src/db.ts [hunk 2]
```diff
@@ -275,3 +276,94 @@ export async function moveAccountDbFromMongoToPG (
...
+            await client`UPDATE ${client(table)} SET "workspaceId" = ${uuid} WHERE "workspaceIdOld" = ${ws.workspace}`
+          }
+        })
+
+        await retryTxn(pgClient, async (client) => {
+          await client`ALTER TABLE ${client(table)} ALTER COLUMN "workspaceId" SET NOT NULL`
+        })
+
+        await retryTxn(pgClient, async (client) => {
+          await client`ALTER TABLE ${client(table)} DROP CONSTRAINT ${client(`${table}_pkey`)}`
```
Comment (BykhovDenis) [inline dev/tool/src/db.ts:359]: As I know this will not work for cockroach, only for PostgreSQL. Have you tested it on cockroach?

Comment (lexiv0re) [reply dev/tool/src/db.ts:359]: Yes, I tested it both on Postgres and Cockroach. It works if you put it in one transaction with new PKEY creation.

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**